packages/agreement-process/src/model/domain/toEvent.ts

```diff
@@ -1,80 +1,10 @@
 import { CreateEvent } from "pagopa-interop-commons";
 import {
   Agreement,
-  AgreementDocument,
-  AgreementStamp,
-  AgreementStamps,
-  AgreementState,
-  AgreementV2,
   AgreementDocumentId,
-  AgreementStateV2,
-  AgreementDocumentV2,
-  AgreementStampV2,
-  AgreementStampsV2,
   AgreementEventV2,
+  toAgreementV2,
 } from "pagopa-interop-models";
-import { match } from "ts-pattern";
-
-export const toAgreementStateV2 = (state: AgreementState): AgreementStateV2 =>
-  match(state)
-    .with("Draft", () => AgreementStateV2.DRAFT)
-    .with("Suspended", () => AgreementStateV2.SUSPENDED)
-    .with("Archived", () => AgreementStateV2.ARCHIVED)
-    .with("Pending", () => AgreementStateV2.PENDING)
-    .with("Active", () => AgreementStateV2.ACTIVE)
-    .with("Rejected", () => AgreementStateV2.REJECTED)
-    .with(
-      "MissingCertifiedAttributes",
-      () => AgreementStateV2.MISSING_CERTIFIED_ATTRIBUTES
-    )
-    .exhaustive();
-
-export const toAgreementDocumentV2 = (
-  input: AgreementDocument
-): AgreementDocumentV2 => ({
-  ...input,
-  createdAt: BigInt(input.createdAt.getTime()),
-});
-
-export const toAgreementStampV2 = (
-  input: AgreementStamp
-): AgreementStampV2 => ({
-  ...input,
-  when: BigInt(input.when.getTime()),
-});
-
-export const toAgreementStampsV2 = (
-  input: AgreementStamps
-): AgreementStampsV2 => ({
-  submission: input.submission
-    ? toAgreementStampV2(input.submission)
-    : undefined,
-  activation: input.activation
-    ? toAgreementStampV2(input.activation)
-    : undefined,
-  rejection: input.rejection ? toAgreementStampV2(input.rejection) : undefined,
-  suspensionByProducer: input.suspensionByProducer
-    ? toAgreementStampV2(input.suspensionByProducer)
-    : undefined,
-  upgrade: input.upgrade ? toAgreementStampV2(input.upgrade) : undefined,
-  archiving: input.archiving ? toAgreementStampV2(input.archiving) : undefined,
-  suspensionByConsumer: input.suspensionByConsumer
-    ? toAgreementStampV2(input.suspensionByConsumer)
-    : undefined,
-});
-
-export const toAgreementV2 = (input: Agreement): AgreementV2 => ({
-  ...input,
-  state: toAgreementStateV2(input.state),
-  createdAt: BigInt(input.createdAt.getTime()),
-  updatedAt: input.updatedAt ? BigInt(input.updatedAt.getTime()) : undefined,
-  suspendedAt: input.suspendedAt
-    ? BigInt(input.suspendedAt.getTime())
-    : undefined,
-  consumerDocuments: input.consumerDocuments.map(toAgreementDocumentV2),
-  contract: input.contract ? toAgreementDocumentV2(input.contract) : undefined,
-  stamps: toAgreementStampsV2(input.stamps),
-});
```

Comment:
Moved these functions to commons `protobufConverterToV2.ts` so I can use them in the new service tests

Reply:
If you base this PR on #373 you could find those already there, correct?

Reply:
True, didn't notice it! I will rebase on that one